Remove CSS vendor prefixes (#19715)
mdn/content: files/en-us/learn/tools_and_testing/cross_browser_testing/html_and_css/index.md

> - Mozilla uses `-moz-`

Reviewer: ```suggestion
- Firefox uses `-moz-`
```

> - Chrome/Opera/Safari use `-webkit-`

Reviewer: ```suggestion
- Chrome/Edge/Opera/Safari use `-webkit-`
```

> - Internet Explorer uses `-ms-`

Reviewer: ```suggestion

More prefixes were used in the past: Internet Explorer and early versions of Edge used `-ms-`, and old versions of Opera used `-o`.
```

Here's some examples:

```css
-webkit-transform: rotate(90deg);
-moz-transform: rotate(90deg);
transform: rotate(90deg);
```

> While the transform propert does not require a prefix, you may encounter this old CSS in a codebase. The first line shows a {{cssxref("transform")}} property with a `-webkit-` prefix — this was needed to make transforms work in older versions of Safari and Chrome until the prefix-free feature was supported.

Reviewer 2: ```suggestion
While the `transform` property does not require a prefix, you may encounter this old CSS in a codebase. The first line shows the {{cssxref("transform")}} property with a `-webkit-` prefix — this was needed to make transforms work in older versions of Safari and Chrome until the prefix-free feature was supported.
```

> The second line has a `-moz-` prefix. Also no longer needed. The third one has no prefix. This third version shows the final version of the syntax supported in all evergreen browsers.

Reviewer: ```suggestion
The second line has a `-moz-` prefix, which is also no longer needed. The third one has no prefix. This third version shows the final version of the syntax supported in all evergreen browsers.
```

> Prefixed features were never supposed to be used in production websites — they are subject to change or removal without warning, and cause cross browser issues. This is particularly a problem when developers decide to only use say, the `-webkit-` version of a property — meaning that the site won't work in other browsers. This actually happened so much that other browsers implemented `-webkit-` prefixed versions of several CSS properties. While browsers still support some prefixed property names, property values, and pseudo classes, now experimental features are put behind flags so developers can test them during development.

Reviewer 2: ```suggestion
Prefixed features were never supposed to be used in production websites — they are subject to change or removal without warning and cause cross-browser issues. This is particularly a problem, for example, when developers decide to use only the `-webkit-` version of a property, which implied that the site won't work in other browsers. This actually happened so much that other browser vendors implemented `-webkit-` prefixed versions of several CSS properties. While browsers still support some prefixed property names, property values, and pseudo classes, now experimental features are put behind flags so that web developers can test them during development.
```
